assets/scss/03-organisms/_header-hamburger.scss

```diff
@@ -263,6 +263,7 @@ body.show-menu {
     transition: left 0.3s ease, visibility 0.3s;
     width: 320px;
     z-index: $z-overlay;
+    padding-bottom: 100px;
```

Comment:
@ygannett This is kind of a heads up because I recognize that this solution is arbitrary. I am adding this padding to account for differences in how the height is calculated across devices, specifically post-iPhone X devices that have a taller bottom toolbar that is allowed to overlay the visible viewport space.